app/renderer/components/preferences/payment/disabledContent.js

```diff
@@ -59,7 +58,7 @@ const styles = StyleSheet.create({
     backgroundColor: globalStyles.color.lightGray,
     borderRadius: globalStyles.radius.borderRadiusUIbox,
     padding: '40px',
-    fontSize: paymentStyles.font.regular,
+    fontSize: globalStyles.payments.fontSize.regular,
```

Comment:
If `paymentStyles` would be left, there would remain only [this line](https://github.com/brave/browser-laptop/pull/10238/files#diff-6dd8c50f92b9abd7a2553f4f3a8b1433L9) as other two will be removed. It does not make much sense to have a const for one style, IMO.